contracts/TokenStoreBalances.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity ^0.4.13;
+
+// Check balances for multiple ERC20 tokens in 1 batched request
+// address 0x0 is used to indicate ETH
+// Based on DeltaBalances 0x3e25f0ba291f202188ae9bda3004a7b3a803599a
+
+// Exchange contract Interface
+contract Exchange {
+  function balanceOf(address /*token*/, address /*user*/) public constant returns (uint);
+}
+
+// ERC20 contract interface
+contract Token {
+  function balanceOf(address /*tokenOwner*/) public constant returns (uint /*balance*/);
+  function transfer(address /*to*/, uint /*tokens*/) public returns (bool /*success*/);
+}
+
+contract TokenStoreBalances {
+
+  address public admin;
+
+  function TokenStoreBalances() public {
+    admin = msg.sender;
+  }
+
+  // Fallback function, don't accept any ETH
+  function() public payable {
+    revert();
+  }
+
+  // Limit withdrawals to the contract creator
+  modifier isAdmin() {
+    require(msg.sender == admin);
+    _;
+  }
+
+  // Backup withdraw, in case ETH gets in here
+  function withdraw() external isAdmin {
+    admin.transfer(address(this).balance);
+  }
+
+  // Backup withdraw, in case ERC20 tokens get in here
+  function withdrawToken(address token, uint amount) external isAdmin {
```

Comment:
Let's remove it, not sure why anybody would send tokens to this address. We can remove admin part and constructor completely after that.

Reply:
Done